### DISCUSSION TITLE
What's the stop command of the standalone mode?

### DISCUSSION THREAD

**Discussion by fishfree** [cantaloupe-project/cantaloupe#504] [Q&A] [answered]
By now, I can only find out the process and then kill it to stop Cantaloupe. Is there a script like run.sh to stop it?

**Comment by adolski** [comment]
There is no such script provided, but you could write one. The basic idea is to write the PID to a file at start, and then kill the PID in the file at stop.

**Comment by giancarlobi** [comment, marked as answer]
@fishfree @adolski On Ubuntu I install Cantaloupe as service, you can find here https://devdbopen.byterfly.eu/doku.php?id=stepbystep:baseservices#cantaloupe "Set Cantaloupe as a service " the steps needed, after that you can start/stop Cantaloupe as any other daemon.

**Comment by fishfree** [comment]
@giancarlobi Thank you very much for your generous sharing!
